libs/telemetry/src/main/java/org/opensearch/telemetry/metrics/MetricsTelemetry.java

```diff
@@ -10,12 +10,14 @@
 
 import org.opensearch.common.annotation.ExperimentalApi;
 
+import java.io.Closeable;
+
 /**
  * Interface for metrics telemetry providers
   *
  * @opensearch.experimental
  */
 @ExperimentalApi
-public interface MetricsTelemetry {
+public interface MetricsTelemetry extends MetricsRegistry, Closeable {
```

Comment:
@Gaganjuneja if you by chance need to push one more time (because of flaky test, we could make a change later): 

```suggestion
public interface MetricsTelemetry extends MetricsRegistry {
```

`MetricsRegistry` already implements `Closeable`.

Reply:
Ah!, sure!